atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/priority/NextTableToSweepProviderTest.java

```diff
@@ -89,6 +102,32 @@ public void calculatorReturnsSingleHighestPriority_thenProviderReturnsThatTable(
         thenTableChosenIs(table("table3"));
     }
 
+    @Test
+    public void calculatorReturnsMultiplePriorities_highestPriorityIsLocked_thenProviderReturnsSecondHighest()
+            throws InterruptedException {
+        givenPriority(table("table1"), 30.0);
+        givenPriority(table("table2"), 20.0);
+        givenTableIsLocked("table1");
+
+        whenGettingNextTableToSweep();
+
+        thenTableChosenIs(table("table2"));
+    }
+
+    @Test
+    public void calculatorReturnsMultiplePriorities_allLocked_thenProviderReturnsNothing()
```

Comment:
rename? 2/3 are locked and the last one has sweep priority (which apparently means not to sweep it)

Reply:
yep, we don't sweep a table if the sweep priority is zero.